### DISCUSSION TITLE
Highlights made with Preview.app on macOS don't appear on Sioyek on Linux

### DISCUSSION THREAD

**Discussion by saurabhsharan** [ahrm/sioyek#1238] [Q&A]
I added some highlights to a PDF using Preview.app on a Mac. I then copied that PDF to my Linux machine and opened it with Sioyek, but the highlights aren't showing up. I tried opening the same PDF with Firefox and the highlights showed up so I don't think it's an issue with the .pdf file. Is there a config setting I'm missing in Sioyek to make highlights appear, or are highlights created in Preview.app not viewable by Sioyek (on Linux)?

**Comment by ahrm** [comment]
Can you share the pdf file?

**Reply by saurabhsharan** [reply]
Yup, I just emailed you a link to the file.

**Reply by ahrm** [reply]
The issue seems to be with mupdf version 1.24. With 1.23 the document looks okay (it looks fine in older sioyek versions for example). I don't know if this is a bug with mupdf or they intentionally decided to stop repairing slightly corrupted pdf annotations, either way, there is nothing that I can do on sioyek's side, if there is a bug, it should be reported to mupdf devs.

**Reply by saurabhsharan** [reply]
Gotcha, thanks a lot for taking a look at this!
